robot_calibration/include/robot_calibration/capture/depth_camera.h

```diff
@@ -37,21 +37,13 @@ class DepthCameraInfoManager
 
   bool init(ros::NodeHandle& n)
   {
-    camera_info_subscriber_ = n.subscribe("/head_camera/depth/camera_info",
+    std::string topic_name = n.getNamespace() +"/camera_info";
+
+    camera_info_subscriber_ = n.subscribe(topic_name,
                                           1,
                                           &DepthCameraInfoManager::cameraInfoCallback,
                                           this);
 
-    // Get parameters of drivers
-    if (!n.getParam("/head_camera/driver/z_offset_mm", z_offset_mm_) ||
-        !n.getParam("/head_camera/driver/z_scaling", z_scaling_))
```

Comment:
the only difference this bit will make is that it will assume that z_offset_mm_ and z_scaling_ to be the default values 0 and 1 respectively. But it will still calibrate these two on fetch. So basically if we reset the calibration(which we do anyway) before calibrating it should work fine. The only reason i deleted this was because these two params don't exist for pmd. Once i find a way around , I will fix this.